spec/std/callstack_spec.cr

```diff
@@ -1,68 +1,62 @@
 require "./spec_helper"
 
-describe "Backtrace" do
-  it "prints file line:colunm" do
-    with_tempfile("compiler_spec_output") do |path|
-      sample = datapath("backtrace_sample")
+private def compile_and_run_file(source_file)
+  with_tempfile("executable_file") do |executable_file|
+    puts executable_file
+    Process.run("bin/crystal", ["build", "--debug", "-o", executable_file, source_file])
+    File.exists?(executable_file).should be_true
+
+    {% if flag?(:darwin) %}
+      Process.run("dsymutil", ["--flat", executable_file])
```

Comment:
Now that I think about it, this might not be needed anymore because IIRC we always run dsymutil in mac now

Reply:
I was not sure about this either, but don't have a mac for testing. I'll remove it.